presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java

```diff
@@ -279,6 +279,7 @@ protected final OperatorFactory createHashProjectOperator(int operatorId, PlanNo
     {
```

Comment:
Can you add a comment in the commit message describing the motivation for this change?